_i18n/en/resources/user-guides/join-monero-matrix.md

```diff
@@ -1,34 +1,35 @@
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
-Monero developers primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
+Most Monero developers primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
 
-In order to access the Matrix protocol you will need to use a Matrix client. A popular client is Element. Element is a free, open-source program that can be accessed via web, desktop, and mobile.
+In order to access the Matrix protocol you will need to use a Matrix client. In this tutorial we will use Element as a client. Element is a free, open-source program that can be accessed via web, desktop, and mobile.
 
 ### Getting Element
 
 Go to [element.io](https://element.io/get-started#download) to download the latest version of the Element app on your computer. Alternatively you can use the Element [web app](https://app.element.io). There are Android apps available on [F-Droid](https://f-droid.org/packages/im.vector.app/) or [Google Play](https://play.google.com/store/apps/details?id=im.vector.app). As well as an app for [iPhone](https://apps.apple.com/app/vector/id1083446067). Once you have accessed the client of your choice create an account on our host server.
 
 ### Setting the custom host server
 
-While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted by the Monero core team to support decentralization. If you do choose this option remember to set your custom server when logging in on a new device.
+While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted by the Monero community members to support decentralization. If you do choose this option remember to set your custom server when logging in on a new device.
 
-When signing up choose to edit your host server and type in `matrix.monero.social` as shown.
+When signing up choose to edit your host server and type in `monero.social` as shown.
```

Comment:
Is this confirmed to be working?

Reply:
I registered my own account using `monero.social`, so it should be like this for a long time.

Reply:
I thought i did mine like that too, but recently (iirc) dan miller said that it wasnt working. 

so now i'm not sure 😅

Reply:
monero.social should be fine (but registrations are temporarily disabled due to spam)

Reply:
Ok thanks